src/lio/bin/lio_warm.c

```diff
@@ -500,7 +500,7 @@ int main(int argc, char **argv)
                 value = tbx_inip_ele_get_value(ele);
                 if (strcmp(rkey, "rid_key") == 0) {
                     tbx_inip_group_free(ig);
-                    tbx_inip_group_set(ig, value);
+                    tbx_inip_group_set(ig, strdup(value));
```

Comment:
The optional summary table prints the RID instead of the data service 
key (host:port/RID) in the first field.  This is the same behavior that 
occurs for any RIDs the resource service doesn't know about.